scripts/rate-limit-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# A script to test rate limiting
+# https://lincolnloop.com/blog/rate-limiting-nginx/
+
+#DOMAIN = "https://dev.trustroots.org"
+
+echo ""
+echo "Testing rate limiting..."
+
+for i in {0..20}; do
+  echo "Attempt $i..."
+  (curl -Is https://dev.trustroots.org/api/auth/signin | head -n1 &) 2>/dev/null; 
+done
```

Comment:
Oh, I didn't realise I pushed this here too. :-D

The actual implementation is done at Nginx but it's useful to have this script around, so I'll let it be here.